nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/hosted/HelmHostedFacet.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2018-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.helm.internal.hosted;
+
+import java.io.IOException;
+
+import org.sonatype.nexus.repository.Facet;
+import org.sonatype.nexus.repository.Facet.Exposed;
+import org.sonatype.nexus.repository.view.Content;
+import org.sonatype.nexus.repository.view.Payload;
+import org.sonatype.repository.helm.internal.AssetKind;
+
+/**
+ * Helm Hosted Facet
+ *
+ * @since 0.0.2
```

Comment:
Q: Was this moved from somewhere?  Seems an older since.

Reply:
yep. copy/paste error. Fixed 5c18845436092fd10ac264980135f8b2edb04ce4